src/SDKs/_metadata/containerregistry_resource-manager.txt

```diff
@@ -0,0 +1,11 @@
+2020-03-25 06:26:44 UTC
```

Comment:
This metadata file is in a wrong place and in an old format.
Please [update build tools](https://github.com/Azure/adx-documentation-pr/blob/master/engineering/adx_netsdk_process.md#updating-build-tools) and generate again. Thanks